app/schema.graphql

```diff
@@ -15,8 +33,25 @@ type AppConfig {
   useFrameNumber: Boolean!
 }
 
+scalar BSON
+
 scalar BSONArray
 
+type BooleanAggregation implements Aggregation {
+  count: Int!
+  exists: Int!
+  false: Int!
+  true: Int!
+}
+
+union BooleanAggregationDataAggregationIntAggregationFloatAggregationRootAggregationStringAggregation =
```

Comment:
This is where this stops making any sense.

Reply:
Maybe add a comment as a reminder why it’s this way

Reply:
This is generated code. I will look into naming unions explicitly in strawberry